backends/advanced/src/advanced_omi_backend/models/waveform.py

```
@@ -0,0 +1,47 @@
+        description="When this waveform was generated"
+    )
+    processing_time_seconds: Optional[float] = Field(
+        None,
+        description="Time taken to generate waveform"
+    )
+
+    class Settings:
+        name = "waveforms"
+        indexes = [
+            "conversation_id",  # Unique lookup by conversation
+        ]
```

Ruff (0.14.11)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)